TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -219,6 +219,66 @@ const BenchmarkMetricRegex = {
                 funcName: geomean,
             }
         }
+    },
+    quarkus_startup: {
+        outerRegex: /Quarkus Test:  measure \d*? ([\s\S\n]*)/,
+        metrics: {
+            "Footprint": {
+                //Example: Footprint for  warmup 0 10129332
+                regex:/Footprint for  measure \d*? (\d*\.?\d*)/,
+                higherbetter: false,
+                units: "kb",
+                },
+            "Startup time in ms": {
+                //Example: Startup time for measure 0 : 53232
+                regex:/Startup time for measure \d*? : (\d*\.?\d*)/,
+                higherbetter: false,
+                units: "ms",   
+            }
+        }
+    },
+    pingperf_baremetal: {
+        outerRegex:/Running \d*? measures([\s\S\n]*)/,
+        metrics: {
+            "Baremetal throughput": {
+                //Example: Requests/sec: 100522.45/
+                regex:/Requests\/sec: (\d*\.?\d*)/,
+                higherbetter: true,
+                units: "Requests/sec"
+            },
+        }
+    },
+    restCrud_quarkus: {
+        outerRegex:/Measure \d*?([\s\S\n]*)/,
+        metrics: {
+            "Throughput": {
+                //Example: Requests/sec: 15145.27
+                regex:/Requests\/sec: (\d*\.?\d*)/,
+                higherbetter: true,
+                units: "Requests/sec",
+            },
+        }
+    },
+    restCrud_multi_user_quarkus: {
+        metrics: {
+            "Throughput": {
+                //Example: Requests/sec: 15145.27  -- in multi user, we are only grabbing the last throughput result - maximal throughput so regex is bit weird.
+                regex:/Measure \d*?[\s\S\n]*Requests\/sec: (\d*\.?\d*)[\s\S\n]*?postgres-quarkus-rest-http-crud/,
+                higherbetter: true,
+                units: "Requests/sec",
+            },
+        }
+    },
+    getting_started: {
+        outerRegex:/Getting-Started Results([\s\S\n]*)/,
+        metrics: {
+            "Throughput": {
+                //Example: "throughput": "6357.32"
+                regex:/"32":[\s\S\n]*?"throughput": "(\d*\.?\d*)"/,
```

Comment:
Example isn't very clear here since regex has "32". If it's hard to show, please leave a comment.